specification/src/main/asciidoc/chapters/Sparkplug_1_Introduction.adoc

```diff
@@ -208,18 +208,19 @@ plantuml::{assetsdir}assets/plantuml/infrastructure-components.puml[format=svg,
 [[introduction_mqtt_servers]]
 ===== MQTT Server(s)
 
-MQTT enabled infrastructure requires that one or more MQTT Servers are present in the
-infrastructure. An MQTT Server must be compatible with the requirements outlined in the
-link:#conformance[Conformance Section]. In addition, it must be sized to properly manage all MQTT
-message traffic.
+Program or device that acts as an intermediary between Clients which publish Application Messages 
+and Clients which have made Subscriptions[MQTTV5-1.2]. MQTT enabled infrastructure requires that one 
```

Comment:
Should this also include MQTT v3.1.1?

Reply:
Since the section numbers are the same in both versions, @wes-johnson, I decided to quote the most recent one to streamline the text. Naturally, we could mention both if you prefer.

Reply:
I'm fine with this